examples/ota-requestor-app/efr32/src/main.cpp

```diff
@@ -117,7 +106,7 @@ __attribute__((used)) ApplicationProperties_t sl_app_properties = {
 
       /// Bitfield representing type of application
       /// e.g. @ref APPLICATION_TYPE_BLUETOOTH_APP
-      .type = APPLICATION_TYPE_ZIGBEE,
+      .type = APPLICATION_TYPE_THREAD,
```

Comment:
What happen if wifi ?

Reply:
So far the ota-requestor-app on EFR32 has only supports Thread. When WiFi support is added this can be updated